src/index.js

```diff
@@ -100,13 +110,11 @@ const MicroModal = (() => {
     addEventListeners () {
       this.modal.addEventListener('touchstart', this.onClick)
       this.modal.addEventListener('click', this.onClick)
-      document.addEventListener('keydown', this.onKeydown)
```

Comment:
The `keydown` listener has been moved to the document level instead of at the individual modal